src/lib.rs

```diff
@@ -993,6 +1001,48 @@ where
     }
 }
 
+#[cfg(feature = "std")]
+impl<T> AsDecimal for Ratio<T> where
+    T: fmt::Display
+        + Clone
+        + Eq
+        + FromPrimitive
+        + Integer
+        + Pow<usize, Output = T>
+        + Signed
+        + std::string::ToString,
```

Comment:
This is implied by `Display`.
(via `impl<T> ToString for T where T: Display`)

Reply:
Got rid of it from the type traits, but needed to add `use std::string::ToString;` elsewhere to make it compile.